### ISSUE TITLE
Issue with PascalCase

### ISSUE THREAD

**Issue by farnabaz** [unjs/scule#4]
There is an unexpected result When I try to convert PascalCase string into kebab-case 

<img width="414" alt="Screen Shot 2021-08-17 at 2 33 42 PM" src="https://user-images.githubusercontent.com/2047945/129707096-716dfafc-b068-4a5b-bf26-693c9c711b1d.png">

```js
// wrong behavior with PascalCase
scule.kebabCase('ALink')
'alink' // wrong, it should be a-link

// If I convert it to camelCase works fine
scule.kebabCase(scule.camelCase('ALink'))
'a-link'

``

**Comment by pi0** [comment]
It is an intended and documented behavior ([test](https://github.com/unjs/scule/blob/main/test/scule.test.ts#L12)) to split on rising edges otherwise `FOOBAR` will be `f-o-o-b-a-r`. When using `camelCase` it will be converted to `aLink` which is `a-link` in kebab format. I'm okay with also doing it for falling edges but needs a major version.

**Comment by pi0** [comment]
Fixed in 0.3.0 (bb942369b2548b995aefc306a0e0f3fbba923426)
